redash/handlers/queries.py

```diff
@@ -228,7 +228,7 @@ def post(self):
             'object_type': 'query'
         })
 
-        return QuerySerializer(query).serialize()
+        return QuerySerializer(query, with_visualizations=True).serialize()
```

Comment:
This will be needed when we implement new query save and open visualization editor the right way.